Source/pack.cpp

```diff
@@ -61,7 +61,7 @@ void PackPlayer(PkPlayerStruct *pPack, int pnum, bool manashield)
 	pPack->targx = pPlayer->_ptargx;
 	pPack->targy = pPlayer->_ptargy;
 	strcpy(pPack->pName, pPlayer->_pName);
-	pPack->pClass = pPlayer->_pClass;
+	pPack->pClass = static_cast<Sint8>(pPlayer->_pClass);

```

Comment:
isn't this unsigned?

Reply:
No:

https://github.com/diasurgical/devilutionX/blob/ced74f76f92bb00064c5c4ffc813fcca31715a70/Source/pack.h#L39

Reply:
Oh, well guess it doesn't matter if we support 127 classes or 255 :)